rainier-core/src/main/scala/com/stripe/rainier/core/SBC.scala

```diff
@@ -35,6 +40,10 @@ final case class SBC[L, T](priorGenerators: Seq[Generator[Double]],
       val timeRemaining = timeTaken * (reps - i) / i
       plot(list, bins, i, reps, lower, upper, timeRemaining)
     }
+    println(s"\nnew goldset:")
```

Comment:
I might move this into the `main` since we might be using SBC not from SBCModel

Reply:
Done